io/plugins/eu.esdihumboldt.hale.io.deegree.test/src/eu/esdihumboldt/hale/io/deegree/mapping/MappingWriterTest.groovy

```diff
@@ -35,6 +36,7 @@ class MappingWriterTest {
 	 * Simple test that checks if a configuration can be successfully written.
 	 */
 	@Test
+	@Ignore("ignoring for the moment to run the tests")
```

Comment:
If it is not included in the Tests product we don't need to ignore it I guess.
Alternatively maybe add a reason like "deegree export currently not working due to Java update"?

Reply:
I have added a meaningful message as you proposed.